README.md

```diff
@@ -44,6 +44,16 @@ $ cd /path/to/codeigniter
 $ bin/server.sh
 ```
 
+#### Arguments
+
+You can give arguments like CodeIgniter4 serve command.
```

Comment:
This is BC break.
See the current usage: <https://github.com/kenjis/codeigniter-composer-installer/pull/34>

Why do we need to break the compatiblity?

Reply:
> Why do we need to break the compatiblity?

CodeIgniter4 not compatible CodeIgniter3.

Do I not need to support document root?
(CodeIgniter4 serve can not change the document root)

Reply:
> CodeIgniter4 not compatible CodeIgniter3.

It is not enough reasoning to break the compatibility.
The command is from this project.

> Do I not need to support document root?

Yes. I have no reasons to break.

Reply:
fixed!